resources/assets/js/features/bookings/components/BookingsContainer.js

```diff
@@ -18,7 +19,9 @@ class BookingsContainer extends React.Component {
             limit: 10,
             errors: {},
             preloader: false,
-            canceledBooking: null
+            canceledBooking: null,
+            reviewBookingModal: null,
```

Comment:
if you're going to use it as bool, set default value to false and rename prop so it could reflect data type that it contains

Reply:
@TatsuUkraine fixed it